src/here-xyz.ts

```diff
@@ -1895,90 +1920,92 @@ async function configXyzSpace(id:string, options:any) {
             }
             schemaDef = schemaDef.replace(/\r?\n|\r/g, " ");
             //console.log(JSON.stringify(schemaDef));
-            let processors = [];
-            processors.push(getSchemaProcessorProfile(schemaDef));
-            patchRequest['processors'] = processors;
+            
+            if(!spacedef.processors)
+                spacedef.processors = [];
+            spacedef.processors.push(getSchemaProcessorProfile(schemaDef));
+            
         }
+        patchRequest['processors'] = spacedef.processors;
     }
-        
-    
-    if(Object.keys(patchRequest).length > 0) {
 
-        if(options.stats) {
+    if (Object.keys(patchRequest).length > 0) {
+
+        if (options.stats) {
             console.log("Request of stats will be ignored, stats option can only be used standalone or with -r/--raw")
         }
 
         const url = `/hub/spaces/${id}?clientId=cli`
         const { response, body } = await execute(
-                url,
-                "PATCH",
-                "application/json",
-                patchRequest,
-                null,
-                false
-            );
+            url,
+            "PATCH",
+            "application/json",
+            patchRequest,
+            null,
+            false
+        );
 
-        if(response.statusCode >= 200 && response.statusCode < 210) {
+        if (response.statusCode >= 200 && response.statusCode < 210) {
```

Comment:
please handle failure scenario

Reply:
hi, the failure scenario are already thrown as errors and handled by handleError method which is part of the same file.